### PR TITLE
Remove uses of deprecated gtk3 apis

### PR DESCRIPTION
**Pull request by deltragon**
Necessary for migration to gtk4.

### CODE REVIEW

File: safeeyes/plugins/trayicon/plugin.py [hunk 1]
```diff
@@ -76,10 +76,7 @@ def __init__(self, context, plugin_config):
         self.menu = Gtk.Menu()
 
         # Next break info menu item
-        self.item_info = Gtk.ImageMenuItem()
-        img_timer = Gtk.Image()
-        img_timer.set_from_icon_name("io.github.slgobinath.SafeEyes-timer", 16)
-        self.item_info.set_image(img_timer)
+        self.item_info = Gtk.MenuItem()
```
Comment (deltragon) [inline safeeyes/plugins/trayicon/plugin.py:79]: Unfortunately, gtk4 completely removes the option of having a MenuItem with an image.
However, in https://github.com/slgobinath/SafeEyes/pull/558 I propose to drop this code entirely.

**Comment by deltragon** [comment]
Note that their are multiple uses of deprecated apis in the trayicon plugin.
However, I propose to completely replace that in https://github.com/slgobinath/SafeEyes/pull/558, as libappindicator is not compatible with gtk4 anyways.

**Comment by deltragon** [comment]
Added a commit to fix deprecations in the delete/reset buttons in the settings dialog.

**Comment by deltragon** [comment]
Would someone be open to reviewing this? I have tested this extensively on my machine, but would prefer having a second set of eyes look over the PR before merging.

**Comment by archisman-panigrahi** [comment]
I can review it. I use plasma 5 (Kubuntu), plasma 6 (arch) and cinnamon (mint).

Where would you like me to review it, and what feature should I review?

**Comment by deltragon** [comment]
I think just the basic features (making sure it starts correctly, the break starts correctly) should be enough.

**Comment by deltragon** [comment]
Oh, and a review on Cinnamon would be good - I've already tested Plasma 5/6.

**Review by archisman-panigrahi — APPROVED** [review]
I tested on Cinnamon, and could not find any regressions. I propose merging.

**Pull request merged**